grabbit/src/test/groovy/com/twcable/grabbit/client/GrabbitConfigurationSpec.groovy

```diff
@@ -25,6 +25,125 @@ import spock.lang.Unroll
 class GrabbitConfigurationSpec extends Specification {
 
     @Unroll
+    def "Should return exclude paths"() {
+        when:
+        def expectedOutput = ["/content/a/b/d", "/content/a/b/e/ab", "/content/a/b/q/fg"] as Collection<String>
+        def actualOutput = GrabbitConfiguration.create(input)
+
+        then:
+        actualOutput instanceof GrabbitConfiguration
+
+        actualOutput.pathConfigurations.excludePaths.get(0) == expectedOutput
```

Comment:
Logically what you're really doing is selecting the first value from `pathConfigurations` and then getting its `excludePaths` value. So it *should* read
```groovy
actualOutput.pathConfigurations.first().excludePaths == expectedOutput
```
The reason the current way works is because since you're not selecting a specific member of the `pathConfigurations` collection, it's doing a "spread" and selecting all of its elements that contain `excludePaths`. It's pretty weird behavior coming from a static background, but common in dynamic contexts, such as XPath. Since here's you're using Groovy in its dynamic form, it's doing the spread. However it would be much more clear if you expressed your intent better by selecting the first `pathConfigurations` instead of relying on the result of it doing the spread combinations.